axelrod/moran.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+from collections import Counter
+import random
+
+import numpy as np
+
+from .deterministic_cache import DeterministicCache
+from .match import Match
+from .player import Player
+from .random_ import randrange
+
+
+def fitness_proportionate_selection(scores):
+    """Randomly selects an individual proportionally to score."""
```

Comment:
Sure, but I'm not sure what to add really beyond the name of the function. [Wikipedia](https://en.wikipedia.org/wiki/Fitness_proportionate_selection) has an extensive explanation.

It's sort of a weighted `randrange` -- if you put in a constant vector then it would be `randrange(0, len(scores))`.